package.json

```diff
@@ -28,14 +28,16 @@
     "test:flow": "flow check",
     "test:lint": "eslint 'lib/**/*.js' 'example/**/*.js'",
     "test:e2e:ios": "detox test -c ios",
-    "test:e2e:android": "detox test -c android"
+    "test:e2e:android": "detox test -c android",
```

Comment:
I like the `build:e2e:<platform>` and `test:e2e:<platform>` names. I'll update NetInfo to match this.